rero_ils/modules/items/api_views.py

```diff
@@ -71,7 +71,7 @@ def decorated_view(*args, **kwargs):
         except NotFound as error:
             raise(error)
         except Exception as error:
-            # raise(error)
+            # raise Exception(error)
```

Comment:
If it's commented, should it be removed?

Reply:
No :-) it's always the place for debugging